src/main/xar-resources/data/contentextraction/listings/listing-3.txt

```diff
@@ -1,3 +1,3 @@
 content:get-metadata($binary as xs:base64Binary) as document-node()
 content:get-metadata-and-content($binary as xs:base64Binary) as document-node()
-content:stream-content($binary as xs:base64Binary, $paths as xs:string*, $callback as function, $namespaces as element()?, $userData as item()*) as empty-sequence()
+content:stream-content($binary as xs:base64Binary, $paths as xs:string*, $callback as function, $namespaces as element()?, $userData as item()*) as empty()
```

Comment:
`empty()` should be `empty-sequence()`

Reply:
indeed something is foul in the state of denmark

Reply:
closing the OR? 😱😳

Reply:
yes the big one will have to wait another round